src/main/java/frc/robot/constants/FieldConstants.java

```diff
@@ -1,9 +1,21 @@
 package frc.robot.constants;
 
+import frc.utils.mirrorutils.MirrorableRotation2d;
+import frc.utils.mirrorutils.MirrorableTranslation3d;
+
 public class FieldConstants {
 
     public static final double FIELD_LENGTH_METERS = 16.54175;
 
     public static final double FIELD_WIDTH_METERS = 8.0137;
 
+    public static final MirrorableTranslation3d SPEAKER = new MirrorableTranslation3d(
+            0.23,
+            FIELD_WIDTH_METERS - 2.55,
+            2.045,
+            true
+    );
+
+    public static final MirrorableRotation2d ANGLE_TO_AMP = MirrorableRotation2d.fromDegrees(90, false);
+
```

Comment:
Why is this false for mirroring? You'd want to face the other way when you're on the red side.

Reply:
its more code that runs, and mirror does nothing so i thought to give up on setting true